.github/workflows/build.yml

```diff
@@ -24,10 +24,7 @@ jobs:
         dnf -y install git cmake ninja-build clang gcc gcc-c++ libstdc++-static
         dnf -y clean all
 
-    # To enable submodules, git must be installed before actions/checkout@v3
     - uses: actions/checkout@v4
```

Comment:
can you move this step above the "Install prerequisites", please ? I had to put it in this non-standard position to get `git` installed first, but now we do not need this unusual order anymore.

Which reminds me - I guess we can also remove `git` installation in the other step.

Reply:
This does not work because FetchContent uses git under the hood.. i put it back now.